dinov2/data/datasets/patch_dataset.py

```diff
@@ -0,0 +1,65 @@
+import glob
+import json
+import os
+
+import pyvips
+from PIL import Image
+from tqdm import tqdm
+from torch.utils.data import Dataset
+
+
+class PatchDataset(Dataset):
```

Comment:
I've modified this class to accept patches with and without saved `.webp` images